scripts/clang_tidy.sh

```diff
@@ -16,7 +16,7 @@ SCRIPT_NAME=$0
 FIX=
 
 function run() {
-  nix develop --command "$@"
+  nix develop ".#bpftrace-llvm19" --command "$@"
```

Comment:
Mind following up with clang-tidy fixes to drop this?

Reply:
I can definitely take a look. The `readability-container-contains` fixes are fine and worth taking, but the `modernize-use-integer-sign-comparison` honestly seem wrong. For example:

    ap.address > static_cast<uint64_t>(arch::max_arg())

This current code is fine. `ap.address` is a `uint64_t`, and the cast on the right is nice and explicit. The recommended fix is:

    std::cmp_greater(ap.address, arch::max_arg())

This is supposed to always do the right thing, but it's different behaviour. I think it would be okay in new code but this isn't a clear fix in any case. What do you think? I'm a bit worried about taking them when not familiar with the code, though @ttreyer  is right that a lot of these go away if you correct the type of a loop variable.

Reply:
Agreed that `std::cmp_greater()` stuff is much less readable. I think it'd be better to avoid the signed-unsigned comparison altogether if possible. And if this lint is too noisy, add it to the block list.